shared/chat/conversation/list/container.js

```diff
@@ -83,21 +80,22 @@ const convStateProps = createSelector(
   })
 )
 
+// TODO this is temp until we can discuss a better solution to this getMessageFromMessageKey thing
+let _stateHack
```

Comment:
getMessageFromMessageKeyFnSelector is making a new closure each time which really kills the hierarchy. really we should dispatch some action which handles this and plumb the data back down. That's a bigger change so i'm doing this short term